cpp/cmake/thirdparty/get_raft.cmake

```diff
@@ -42,6 +42,8 @@ set(CUGRAPH_BRANCH_VERSION_raft "${CUGRAPH_VERSION_MAJOR}.${CUGRAPH_VERSION_MINO
 # To use a different RAFT locally, set the CMake variable
 # RPM_raft_SOURCE=/path/to/local/raft
 find_and_configure_raft(VERSION    ${CUGRAPH_MIN_VERSION_raft}
-                        FORK       rapidsai
-                        PINNED_TAG branch-${CUGRAPH_BRANCH_VERSION_raft}
+                        # FORK       rapidsai
+                        # PINNED_TAG branch-${CUGRAPH_BRANCH_VERSION_raft}
+                        FORK       trxcllnt
+                        PINNED_TAG fix/build-shared-faiss
```

Comment:
I'm assuming this is just for debugging the CI failures?

Reply:
Without this, CI in this PR would never go green :-)